Doc adding more graphviz commands (#18502)
tldr-pages/tldr: pages/common/gvgen.md

# gvgen

> Generate simple, structured abstract graphs.
> More information: <https://graphviz.org/pdf/gvgen.1.pdf>.

> - Generate a cycle with 10 vertices and edges:

Reviewer: ```suggestion
- Generate a [c]ycle with 10 vertices and edges:
```

`gvgen -c {{10}}`

> - Generate a 4×3 grid:

Reviewer: ```suggestion
- Generate a 4×3 [g]rid:
```

`gvgen -g {{4,3}}`

> - Generate a binary tree of height 5:
> 
> `gvgen -t {{5}}`
> 
> - Generate a complete bipartite graph with 3 and 4 vertices:
> 
> `gvgen -b {{3,4}}`
> 
> - Create a random graph and write it to a file:
> 
> `gvgen -r {{10,5}} -o {{random.gv}}`
> 
> - Generate a directed graph with verbose output:

Reviewer: ```suggestion
- Generate a binary [t]ree of height 5:

`gvgen -t {{5}}`

- Generate a complete [b]ipartite graph with 3 and 4 vertices:

`gvgen -b {{3,4}}`

- Create a [r]andom graph and [o]utput it to a file:

`gvgen -r {{10,5}} -o {{random.gv}}`

- Generate a [d]irected graph with [v]erbose output:
```

`gvgen -d -v -c {{6}}`

> - Display usage information:

Reviewer: ```suggestion
- Display help:
```

# gvgen

> Generate simple, structured abstract graphs.

> > Supports cycles, grids, trees, meshes, and other standard graph types.

Reviewer: ```suggestion
```
